### ISSUE TITLE
Gone from Google Play Store. Is the app dead?

### ISSUE THREAD

**Issue by drspy00** [segler-alex/RadioDroid#1236]

**Comment by Andre0be** [comment]
Probably

**Comment by doomrunningonyomamma** [comment]
(⁠ ⁠･ั⁠﹏⁠･ั⁠) Save us Morcox

**Comment by splisp** [comment]
@segler-alex is quite active on the new home for the RadioBrowser project on GitLab (https://gitlab.com/users/segler_alex/activity) but it seems he hasn't worked on RadioDroid for a long time :(

**Comment by morckx** [comment]
I've just released an updated debug build with support for Android Auto (which is probably hard to maintain on for Play Store builds) and disabled battery optimization in [my fork here](https://github.com/morckx/RadioDroid/releases).

**Comment by splisp** [comment]
> I've just released an updated debug build with support for Android Auto (which is probably hard to maintain on for Play Store builds) and disabled battery optimization in [my fork here](https://github.com/morckx/RadioDroid/releases).

Great, I'll check it out in the car.  I might also try building from your fork. Do you plan to also support Google TV platform? That would be awesome!

**Comment by morckx** [comment]
> Great, I'll check it out in the car. I might also try building from your fork. Do you plan to also support Google TV platform? That would be awesome!

Google TV should also work. However, I checked it so far only with older Android TV versions, and there might be some color issues, as I might have accidentally overwritten some color changes, [here](https://github.com/morckx/RadioDroid/commit/582196effa96c45de8e234d8f80eed83f9f8a2dd).

**Comment by splisp** [comment]
@morckx on your fork there's no Issues page; I have tried your latest build on a Google TV device and I wanted to report my findings (it's not usable but the issues are limited).

**Comment by morckx** [comment]
> @morckx on your fork there's no Issues page; I have tried your latest build on a Google TV device and I wanted to report my findings (it's not usable but the issues are limited).

OK, I've activated issues on my fork and tried it now on Google TV, too. My main issues are: "Bottom navigation" should definitely be deactivated on Google TV, and some focus colors are not really visible.

**Comment by STPKITT** [comment]
> I've just released an updated debug build with support for Android Auto (which is probably hard to maintain on for Play Store builds) and disabled battery optimization in [my fork here](https://github.com/morckx/RadioDroid/releases).

There already is an option "Battery optimization" in the original though. It's in the setup -> Misc or Other or whatever it's called on devices running the English language.

**Comment by morckx** [comment]
> > I've just released an updated debug build with support for Android Auto (which is probably hard to maintain on for Play Store builds) and disabled battery optimization in [my fork here](https://github.com/morckx/RadioDroid/releases).
> 
> There already is an option "Battery optimization" in the original though. It's in the setup -> Misc or Other or whatever it's called on devices running the English language.

Right, thanks.  My PR https://github.com/segler-alex/RadioDroid/pull/1125 was merged some time ago.

**Comment by STPKITT** [comment]
:-) Thanks very much for your work on this app!

**Comment by ONDEBAY** [comment]
El nuevo fork de RadioDroid está aquí: - The new fork of RadioDroid is here:
https://github.com/xueshi86/RadioDroid
